src/main/resources/com/google/api/codegen/java/operation_callable.snip

```diff
@@ -0,0 +1,49 @@
+@extends "java/common.snip"
+
+@snippet renderStubFileHeader(fileHeader)
+  {@license(fileHeader)}
+  package {@fileHeader.packageName};
+
+  {@importList(fileHeader.importSection.appImports)}
+@end
+
+@snippet generate(classFile)
+  {@renderStubFileHeader(classFile.fileHeader)}
+
+  {@classDoc()}
+  @@Generated("by gapic-generator")
+  @@BetaApi("The surface for long-running operations is not stable yet and may change in the future.")
+  {@renderClass(classFile.classView)}
+@end
+
+@private classDoc()
+  // AUTO-GENERATED DOCUMENTATION AND CLASS
+  /**
+   * A {@@code UnaryCallable} that wraps a UnaryCallable returning a
+   * long-running operation API resource and returns an OperationSnapshot instead.
+   */
+@end
+
+@private renderClass(classView)
+  public class OperationSnapshotCallable<RequestT>
```

Comment:
Do discogapic client classes have prefixes, indicating that they belong to discogapic client? For example, the analogous class in grpc is called `GrpcOperationSnapshotCallable`. There is a fundamental difference between the classes here of course (that `GrpcOperationSnapshotCallable` is api-independent, and shared by all clients, while this one is generated per discogapic client). Should we name this class `<ServiceName>OperationSnapshotCallable`? With current name, the class seems like a class, that `GrpcOperationSnapshotCallable` would want to extend (has more generic name)